src/cnd/axios_rfc7807_middleware.ts

```diff
@@ -7,6 +7,7 @@ export class Problem extends Error {
   public readonly status?: number;
   public readonly detail?: string;
   public readonly instance?: string;
+  public readonly isProblem: boolean;
```

Comment:
From your PR description I take that you use this field to identify whether we have an `error` or a `problem`. If `Problem.isProblem` is always true we can assume it is always true and don't have to check for it right? Instead we can check if the `type of` the received exception is `Problem`. 

e.g. 

```
try {
	swap.redeem();
} catch (error) {
	if (error instanceof ChainError) {
		// do something
    } else if (error instanceof Problem) {
       // do something else
    } else {
      panicAtTheDiscotheque
	}
}
```

What do you think about this?

Reply:
Oh, @thomaseizinger had the same comment :)

Reply:
Stop reading my mind @bonomat, that is scary!

Reply:
Didn't have to. When reviewing code the voice in my head sounds like you 🙉🙊🙈